src/ucp/core/ucp_ep.inl

```diff
@@ -25,19 +25,28 @@ static inline ucp_ep_config_t *ucp_ep_config(ucp_ep_h ep)
 }
 
 static UCS_F_ALWAYS_INLINE uct_ep_h ucp_ep_get_lane(ucp_ep_h ep,
-                                                    ucp_lane_index_t lane_index)
+                                                    ucp_lane_index_t lane_index,
+                                                    int fast_path)
 {
     ucs_assert(lane_index != UCP_NULL_LANE);
 
-    return ep->uct_eps[lane_index];
+	if (fast_path || (lane_index < UCP_MAX_FAST_PATH_LANES)) {
+		return ep->uct_eps[lane_index];
+    }
+	
+	return ep->ext->uct_eps[lane_index - UCP_MAX_FAST_PATH_LANES];
 }
 
 static UCS_F_ALWAYS_INLINE void ucp_ep_set_lane(ucp_ep_h ep, size_t lane_index,
                                                 uct_ep_h uct_ep)
 {
     ucs_assert(lane_index != UCP_NULL_LANE);
 
-    ep->uct_eps[lane_index] = uct_ep;
+	if (lane_index < UCP_MAX_FAST_PATH_LANES) {
```

Comment:
check clang-format

Reply:
no clang-format errors in this file, what caught your attention?

Reply:
indent by 8